arangod/Cache/FrequencyBuffer.h

```diff
@@ -122,18 +109,21 @@ class FrequencyBuffer {
     for (auto f : frequencies) {
       data.emplace_back(std::pair<T, std::size_t>(f.first, f.second));
     }
-    std::sort(data.begin(), data.end(),
-              [](std::pair<T, std::uint64_t> const& left,
-                 std::pair<T, std::size_t> const& right) {
-                return left.second < right.second;
-              });
+    std::sort(
+        data.begin(), data.end(),
+        [](std::pair<T, std::uint64_t> const& left,
+           std::pair<T, std::size_t> const& right) {
```

Comment:
Well, It's definitely doesn't look good. It probably should be
```suggestion
        [](std::pair<T, std::uint64_t> const& left,
           std::pair<T, std::uint64_t> const& right) {
```

Reply:
Good catch. I actually did not change this part, but I also didn't see it.
Fixed.